Client/Application/Delegates/AppDelegate.swift

```diff
@@ -624,21 +628,76 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
             return true
         }
 
-        // Otherwise, check if the `NSUserActivity` is a CoreSpotlight item and switch to its tab or
-        // open a new one.
-        if userActivity.activityType == CSSearchableItemActionType {
-            if let userInfo = userActivity.userInfo,
-                let urlString = userInfo[CSSearchableItemActivityIdentifier] as? String,
-                let url = URL(string: urlString) {
-                browserViewController.switchToTabForURLOrOpen(url, isPrivileged: true)
+        switch userActivity.activityType {
+            case CSSearchableItemActionType:
+                // Otherwise, check if the `NSUserActivity` is a CoreSpotlight item and switch to its tab or
+                // open a new one.
+                if let userInfo = userActivity.userInfo,
+                    let urlString = userInfo[CSSearchableItemActivityIdentifier] as? String,
+                    let url = URL(string: urlString) {
+                    browserViewController.switchToTabForURLOrOpen(url, isPrivileged: true)
```

Comment:
`isPrivileged` set to true can access local-urls. Do we need this to be set to true here?

Reply:
Yes you are right, we should not have have `isPrivileged` true.